etc/alembic.ini.template

```diff
@@ -1,4 +1,5 @@
-# Copyright 2014-2019 CERN for the benefit of the ATLAS collaboration.
+# -*- coding: utf-8 -*-
+# Copyright CERN since 2014
```

Comment:
The unified header for everything is now:

`Copyright European Organization for Nuclear Research (CERN) since 2012`

Reply:
Thanks Mario, will fix! So, we put `2012` even for code/files written after that date?